src/TLuaInterpreter.h

```diff
@@ -32,15 +32,11 @@
 #include <QTimer>
 #include "post_guard.h"
 
-#ifndef LUA_CPP
```

Comment:
It's something I originally added to allow me to compile lua as C++ rather than C. It turns out that doing that isn't as useful as I thought it was.